src/org/usfirst/frc/team467/robot/Grabber.java

```diff
@@ -60,39 +66,50 @@ public void periodic() {
 		if (!RobotMap.HAS_GRABBER) {
 			return;
 		}
-		
+
 		double speed = 0.0;
+		double moveDist = 0.8;
+
 		switch (state) {
 
 		case GRAB:
 			if(hasCube() || count > grabCount) {
 				state = GrabberState.NEUTRAL;
 			} else {
 				speed = RobotMap.MAX_GRAB_SPEED;
+
+				if(!RobotMap.useSimulator) {
+					drive.moveFeet(moveDist);
+				}
+				drivesimulator.moveFeet(moveDist); 	
```

Comment:
We don't want to drive from the Grabber class. I think we have a new, better way to grab and drive at the same time now. Please take a look at the latest code on master and see if you think we still need more work there.